src/lib/tabs/tab-header.html

```diff
@@ -1,8 +1,11 @@
 <div class="mat-tab-header-pagination mat-tab-header-pagination-before mat-elevation-z4"
+     #previousPaginator
      aria-hidden="true"
      mat-ripple [matRippleDisabled]="_disableScrollBefore || disableRipple"
      [class.mat-tab-header-pagination-disabled]="_disableScrollBefore"
-     (click)="_scrollHeader('before')">
+     (click)="_handlePaginatorClick('before')"
+     (mousedown)="_handlePaginatorPress('before')"
+     (touchend)="_stopInterval()">
```

Comment:
Why the mix of `mousedown` and `touchend`?

Reply:
The `touchend` is there to clear the interval in case the `mousedown` doesn't fire since it behaves closer to a click on touch devices.